test/flow/api/browser-upgrade.js

```diff
@@ -15,10 +15,12 @@ module.exports = function (browser) {
   };
 
   browser.setCurrentPosition = function setCurrentPosition(lat, lon, heading, done) {
-    browser.execute(function (lat2, lon2, heading2) {
+    return browser.execute(function (lat2, lon2, heading2) {
       window.mock.geolocation.setCurrentPosition(lat2, lon2, heading2, true);
     }, [lat, lon, heading], function () {
-      done();
+      if (done) {
```

Comment:
[This one](https://github.com/HSLdevcom/digitransit-ui/pull/713/files#diff-205cce2f04b6491670eaa0f5d5f7e300R62)